pygmt/src/meca.py

```diff
@@ -99,7 +99,7 @@ def data_format_code(convention, component="full"):
     R="region",
     J="projection",
     B="frame",
-    C="offset",
+    A="offset",
```

Comment:
Could you maybe move A above B to make it alphabetically ordered?

Reply:
Done in fca2520e443f6c0b4aeeca5719a725617188e8f6.